lib/src/runner/node/platform.dart

```diff
@@ -141,7 +165,19 @@ class NodePlatform extends PlatformPlugin {
           sdkRoot: p.toUri('packages/\$sdk'));
     }
 
-    return new Pair(await Process.start(_executable, [jsPath]), mapper);
+    return new Pair(await _startProcess(platform, jsPath), mapper);
+  }
+
+  /// Starts the Node.js process for [platform] with [jsPath].
+  Future<Process> _startProcess(TestPlatform platform, String jsPath) async {
+    try {
+    return await Process.start(_settings[platform].executable, [jsPath]);
```

Comment:
Is this formatted correctly? looks like the return should be indented.

Reply:
Oops, forgot to reformat!